manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -441,8 +471,8 @@ func (r *FybrikApplicationReconciler) GetWorkloadCluster(application *api.Fybrik
 		if application.Spec.Selector.WorkloadSelector.Size() == 0 {
 			return multicluster.Cluster{}, nil
 		}
-		// the workload runs in a local cluster
-		r.Log.V(0).Info("selector.clusterName field is not specified for an existing workload - a local cluster is assumed")
+		r.Log.Warn().Err(errors.New("selector.clusterName field is not specified")).Str(logging.FYBRIKAPPUUID, utils.GetFybrikApplicationUUID(application)).Str(logging.ACTION, logging.CREATE).Msg("No workload cluster indicated, so a local cluster is assumed")
```

Comment:
It is not an error. Cluster of the workload does not have to be specified if the workload runs on a local cluster.

Reply:
Agreed.  That's why it's a warning